frontend/src/components/common/Resource/EditorDialog.tsx

```diff
@@ -49,12 +59,12 @@ import SimpleEditor from './SimpleEditor';
 type KubeObjectIsh = Partial<KubeObjectInterface>;
 
 export interface EditorDialogProps extends DialogProps {
-  /** The object to edit, or null to make the dialog be in "loading mode". Pass it an empty object if no contents are to be shown when the dialog is first open. */
-  item: KubeObjectIsh | null;
+  /** The object(s) to edit, or null to make the dialog be in "loading mode". Pass it an empty object if no contents are to be shown when the dialog is first open. */
+  item: KubeObjectIsh | object | object[] | string | null;
```

Comment:
minor nit: If the only string value this prop can accept is 'default' then it should be represented in its' type
`((...args: any[]) => void) | "default" | null`